### PR TITLE
Angular9 content children fix

### PR DESCRIPTION
**Pull request by TomaszCz**
After recent adjustments in how the @ContentChildren - descendants: false flag works
[details](https://github.com/angular/angular/issues/14769#issuecomment-586726706)
The proposed pull request contains change which will allow for nested step directives detection.

<a href="https://gitpod.io/#https://github.com/madoar/angular-archwizard/pull/266"></a>

Issue #265

### CODE REVIEW

**Comment by TomaszCz** [comment]
> @TomaszCz Thank you for providing a fix for the regression!
> The PR is missing a test case to ensure that the same issue won't occur again. Can you please add such a test case?

Done.

File: src/lib/components/wizard.component.spec.ts [hunk 1]
```diff
@@ -17,9 +17,11 @@ import { WizardStep } from '../util/wizard-step.interface';
...
         Step 2
       </aw-wizard-step>
-      <aw-wizard-step stepTitle='Steptitle 3' *ngIf="showStep3">
-        Step 3
-      </aw-wizard-step>
+      <div>
+        <aw-wizard-step stepTitle='Steptitle 3' *ngIf="showStep3">
+            Step 3
+        </aw-wizard-step>
+      </div>
```
Comment (madoar) [inline src/lib/components/wizard.component.spec.ts]: I think it would be create a new dedicated test file instead.
In my opinion this is extended functionality which should not influence the internal functionality tests.
Therefore I think it would be best to create a new test file with two test cases:
- a test case to check whether the component can be created
- a test case to check whether the created wizard has the correct number of steps

Ideally I think the new test setup should consist of multiple `div` containers.

Comment (TomaszCz) [reply src/lib/components/wizard.component.spec.ts]: To clarify - my fix + test case are addressing an angular breaking change, I don't think it is a "extended functionality". Anyway, I have created the test case as you wish, and i hope you will decide to merge this soon.

Comment (madoar) [reply src/lib/components/wizard.component.spec.ts]: > To clarify - my fix + test case are addressing an angular breaking change, I don't think it is a "extended functionality".

By extended functionality I mean this is something that the common user of the library does not use. Correct me if I'm wrong but as far as I know the typical user adds his `aw-wizard-step` elements directly to the `aw-wizard` environment.

> Anyway, I have created the test case as you wish, and i hope you will decide to merge this soon.

I'm doing what I can

File: src/lib/components/wizard.component-nested-steps.spec.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        }).compileComponents();
+    }));
+
+    beforeEach(() => {
+        wizardTestFixture = TestBed.createComponent(WizardTestComponent);
+        wizardTestFixture.detectChanges();
+
+        wizardTest = wizardTestFixture.componentInstance;
+        wizard = wizardTest.wizard;
+    });
```
Comment (madoar) [inline src/lib/components/wizard.component-nested-steps.spec.ts]: Now that #264 has been merged these lines most likely needs to be changed slightly to make it work correctly:

https://github.com/madoar/angular-archwizard/blob/d3f4071451b638b5783f8842d167a7fba50f38ac/src/lib/components/wizard.component.spec.ts#L54-L64

File: src/lib/components/wizard.component.spec.ts [hunk 3]
```diff
@@ -18,7 +18,7 @@ import { WizardStep } from '../util/wizard-step.interface';
         Step 2
       </aw-wizard-step>
       <aw-wizard-step stepTitle='Steptitle 3' *ngIf="showStep3">
-        Step 3
+            Step 3
```
Comment (madoar) [inline src/lib/components/wizard.component.spec.ts]: Please revert this change

Comment (TomaszCz) [reply src/lib/components/wizard.component.spec.ts]: Reverted :)

File: src/lib/components/wizard.component-nested-steps.spec.ts (hunk 2 above)
Comment (TomaszCz) [reply src/lib/components/wizard.component-nested-steps.spec.ts]: Formatting changed to prettier, changed included :)

**Comment by TomaszCz** [comment]
I rebased this branch to latest develop. All tests are passing.
